recipe/meta.yaml

```
@@ -33,15 +33,15 @@ requirements:
   run:
     - python
     - pip >=18.0
-    - anyqt >=0.1.0
-    - pyqt
+    - anyqt >=0.2.0
     - docutils
     - commonmark >=0.8.1
     - requests
-    - cachecontrol-with-filecache >=0.12.6
-    # via cachecontrol[filecache]
+    - requests-cache
     - dictdiffer
-    - qasync
+    - qasync >=0.10.0
+    - packaging
+    - {{ pin_compatible('numpy') }}
```

```suggestion
    - numpy
```
As numpy is not used in host, pin_compatible is not required.